### DISCUSSION TITLE
可否出一个Linux上解压即用的milvus服务的程序包，因为服务器限制了试用docker ，源码编译又太繁琐又会出问题

### DISCUSSION THREAD

**Discussion by gongyafei123** [milvus-io/milvus#33365] [Q&A and General discussion]
可否出一个Linux上解压即用的milvus服务的程序包，因为服务器限制了试用docker ，源码编译又太繁琐又会出问题

**Comment by yhmo** [comment]
看这个是否能行吧，最新的milvus-lite是在pymilvus包里一起发布的，以库的形式运行，而不是以C/S的形式。
https://github.com/milvus-io/milvus-lite

其实我们出这种程序包也一样担心编译麻烦繁琐在各种平台下出问题难解决，这种事情大家都不好干。

**Reply by gongyafei123** [reply]
感谢，我试试，那这种方式也避免了用attu去连接
